client/asset/dcr/dcr_test.go

```diff
@@ -1766,6 +1770,10 @@ func TestFindRedemption(t *testing.T) {
 		},
 		Hex: txHex,
 	}
+	walletTxErr := new(error)
```

Comment:
So later we can set the value inside the function without rewriting the entire function.

But since it is confusing will set the entire function.

The reason for changing to a function is because the new method I added will need two or more walletTx there so this way tests can produce different txn based on some logic.